css/formio.css

```diff
@@ -47,6 +47,11 @@
   border: 1px solid red;
 }
 
+.errorButton button{
```

Comment:
You should not need this, but rather just add the ```btn-danger``` class to the button. 

https://getbootstrap.com/docs/3.3/css/#buttons-options

This way it will look good with all themes.